test/fixtures/r-httr2/get_with_all_numerical_data.r

```diff
@@ -0,0 +1,10 @@
+library(httr2)
+
+request("http://localhost:28139/CurlToNode") |> 
+  req_method("POST") |> 
+  req_headers(`Accept` = "application/json") |> 
+  req_body_raw(
+    "18233982904",
```

Comment:
Given title of fixture, should this be a number instead of a string?

Reply:
We could try to detect it but most other parsers also don't. So, I think it's fine to skip this edge case for now.